core/tauri/src/path/desktop.rs

```diff
@@ -6,7 +6,7 @@ use super::{Error, Result};
 use crate::{AppHandle, Manager, Runtime};
 use std::path::PathBuf;
 
-/// A helper class to access the mobile camera APIs.
+/// The path resolver is a helper class for general and application-specific path APIs.
```

Comment:
```suggestion
/// The path resolver is a helper class to get general and application-specific path APIs.
```
or is that better?

Reply:
`for` here makes more sense with the ending `path APIs` IMO